<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Credit</class>
 <widget class="QWidget" name="Credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1141</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <widget class="QLabel" name="l_loan_amount">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Total loan amount</string>
      </property>
     </widget>
     <widget class="QLabel" name="l_loan_term">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Loan term(month)</string>
      </property>
     </widget>
     <widget class="QLabel" name="l_interest_rate">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>160</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Interest rate</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_loan_amount">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>60</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>10000</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_loan_term">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>110</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>12</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_interest_rate">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>160</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>12</string>
      </property>
     </widget>
     <widget class="QPushButton" name="calculate">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>390</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Calc</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>230</y>
        <width>118</width>
        <height>82</height>
       </rect>
      </property>
      <property name="title">
       <string>Type</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="Differential">
         <property name="text">
          <string>Differential</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="Annuity">
         <property name="text">
          <string>Annuity</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>80</y>
        <width>651</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="l_tip">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>30</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="l_tip_plus_pap">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>30</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="l_name_tip">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>0</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Total interest payment</string>
      </property>
     </widget>
     <widget class="QLabel" name="l_name_tip_plus_pap">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>0</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Total interest payment + principal amount payment</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
